l10n/en-US/viewer.ftl

```diff
@@ -416,3 +416,52 @@ pdfjs-editor-colorpicker-red =
 pdfjs-editor-highlight-show-all-button-label = Show all
 pdfjs-editor-highlight-show-all-button =
     .title = Show all
+
+## New alt-text dialog
+## Group note for entire feature: Alternative text (alt text) helps when people can't see the image. This feature includes a tool to create alt text automatically using an AI model that works locally on the user's device to preserve privacy.
+
+# Modal header positioned above a text box where users can edit the alt text.
+pdfjs-editor-new-alt-text-dialog-edit-label = Edit alt text (image description)
+
+# Modal header positioned above a text box where users can edit the alt text.
+pdfjs-editor-new-alt-text-dialog-add-label = Add alt text (image description)
+
+pdfjs-editor-new-alt-text-textarea =
+    .placeholder = Write your description here…
+
+# This text is refers to the alt text box above this description. It offers a definition of alt text.
+pdfjs-editor-new-alt-text-description = Short description for people who can’t see the image or when the image doesn’t load.
+
+# This is a required legal disclaimer that refers to the automatically created text inside the alt text box above this text. It disappears if the text is edited by a human.
+pdfjs-editor-new-alt-text-disclaimer = This alt text was created automatically.
+pdfjs-editor-new-alt-text-disclaimer-learn-more-url = Learn more
+
+pdfjs-new-alt-text-create-automatically-button-label = Create alt text automatically
+pdfjs-editor-new-alt-text-not-now-button = Not now
+pdfjs-editor-new-alt-text-error-title = Couldn’t create alt text automatically
+pdfjs-editor-new-alt-text-error-description = Please write your own alt text or try again later.
+pdfjs-editor-new-alt-text-error-close-button = Close
+
+# Variables:
+#   $totalSize (Number) - the total size (in Mb) of the ai model.
+#   $downloadedSize (Number) - the downloaded size (in Mb) of the ai model.
+#   $percent (Number) - the percentage of the downloaded size.
+pdfjs-editor-new-alt-text-ai-model-downloading-progress =
+    .aria-valuemin = 0
+    .aria-valuemax = $totalSize
```

Comment:
```suggestion
    .aria-valuemax = { $totalSize }
```

Variables in Fluent are surrounded by `{ }`(also applies to all the other variables in these strings)